src/modules/map/components/WarningRibbon.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div v-if="showWarningRibbon" class="corner-ribbon" data-cy="warning-ribbon">TEST</div>
+</template>
+
+<script>
+import { WARNING_RIBBON_HOSTNAMES, DEV_SITE_WARNING } from '@/config'
+
+export default {
+
+    data() {
+        return {
+           showWarningRibbon: DEV_SITE_WARNING ?
+             false : WARNING_RIBBON_HOSTNAMES.some(hostname=>location.host.includes(hostname))
+        }
```

Comment:
This should be a computed property not a data.

Reply:
I would be more defensive here and always show the warning if the host is not `map.geo.admin.ch` and in case of `test.map.geo.admin.ch` show the corner banner instead of showing the banner on matching host name. This would prevent in future to forgot the banner on any new testing host name.

Reply:
OK, this now very defensive and the big invasive `Dev Site Warning` is also displayed unless on (test.)map.geo.admin.ch host.